vassal-app/src/main/java/VASSAL/tools/menu/MenuScroller.java

```diff
@@ -433,29 +429,6 @@ public void keepVisible(int index) {
     keepVisibleIndex = index;
   }
 
-  /**
-   * Removes this MenuScroller from the associated menu and restores the
-   * default behavior of the menu.
-   */
-  public void dispose() {
```

Comment:
Doesn't this need to be deprecated instead of removed?

Reply:
Theoretically, yes. Practically, is anyone likely to call this?

Reply:
No, but I would prefer not getting bug reports about it.